src/experiments/rl_envs/JuliaRL_REMDQN_CartPole.jl

```diff
@@ -16,7 +16,7 @@ function RLCore.Experiment(
 
     env = CartPoleEnv(; T = Float32, rng = rng)
     ns, na = length(state(env)), length(action_space(env))
-    ensemble_num = 6
+    ensemble_num = 16
```

Comment:
Why do we need to increase the `ensemble_num` here? I think doing so will increase the testing time in CI a lot.

Reply:
My tests found that it doesn't actually have much of an effect.

Reply:
OK